### PR TITLE
enable CD for the repository

### PR DESCRIPTION
**Pull request by jtnord**
see also https://github.com/jenkins-infra/repository-permissions-updater/pull/2106

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,11 +64,11 @@
   </scm>
 
   <properties>
-    <revision>2.12.4-2</revision>
+    <revision>2.12.4</revision>
```
Comment (timja) [inline pom.xml:67]: the second example on the page is the recommended format for library plugins:
https://www.jenkins.io/doc/developer/publishing/releasing-cd/#pom-file-modifications

Also changelist.format change seems to be missing here?

Comment (jtnord) [reply pom.xml:67]: should be fixed in 4af2a67a5327560e768133b9886b2de1ceb97c3b

File: .github/workflows/cd.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0
+    - name: Set up JDK 8
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: 8
+    - name: Release
+      uses: jenkins-infra/jenkins-maven-cd-action@v1.1.0
```
Comment (timja) [inline .github/workflows/cd.yaml:55]: I just merged https://github.com/jenkinsci/.github/pull/57, recently got permissions there =/

```suggestion
      uses: jenkins-infra/jenkins-maven-cd-action@v1.2.0
```

**Review by timja — APPROVED** [review]

**Review by car-roll — APPROVED** [review]

**Review by jglick — APPROVED** [review]
Looks right.

**Comment by jglick** [comment]
https://github.com/jenkinsci/jackson2-api-plugin/releases/tag/2.13.0-226.v0c5dd2d2fd2a looks like it worked.

**Comment by timja** [comment]
Dependabot doesn't seem interested in it for `bom`

I wonder if it can't handle the version format change

**Comment by jtnord** [comment]
> https://github.com/jenkinsci/jackson2-api-plugin/releases/tag/2.13.0-226.v0c5dd2d2fd2a looks like it worked.

dependabot also needed https://github.com/jenkinsci/jackson2-api-plugin/pull/99/files

**Comment by jglick** [comment]
> I wonder if it can't handle the version format change

Not sure. `2.13.0-226.v0c5dd2d2fd2a` seems clearly newer than `2.12.4-1`?

**Comment by timja** [comment]
@jtnord I meant dependabot users consuming this repo, it’s currently refusing to update the bom no matter how much I poke it

I’ve created a PR to update to this version to see if it fixes the next one https://github.com/jenkinsci/bom/pull/669

**Pull request merged**